modules/end-user-guide/partials/proc_creating-a-workspace-by-importing-the-source-code-of-a-project.adoc

```diff
@@ -107,35 +148,54 @@ image::workspaces/configure-command.png[Configure tasks]
 
 
 [id="importing-to-a-running-workspace-using-the-git-clone-command_{context}"]
-== Importing to a running workspace using the *Git: Clone* command
+== Importing a project with *Git: Clone* in the GUI
+
+You can import your project into a running workspace with the *Git: Clone* command in the graphic user interface of {prod-short}.
+ 
+.Prerequisites
+* A started workspace
```

Comment:
.

Reply:
Period omitted deliberately. IBM Style Guide: p. 89 "_Write lists so that all or none of the items start with complete sentences. ... If list items comprise only sentence fragments, do not include any end punctuation_."